algorithmia/manifest.json

```
@@ -12,8 +12,7 @@
     "title": "Algorithmia",
     "media": [],
     "classifier_tags": [
-      "Supported OS::Linux",
-      "Category::Monitoring"
+      "Supported OS::Linux"
```

```suggestion
      "Category::Metrics",
      "Supported OS::Linux"
```
Adding metrics for now